src/Driver/Driver.php

```diff
@@ -539,24 +538,17 @@ protected function bindParameters(
     protected function formatDatetime(DateTimeInterface $value): string
     {
         try {
-            $datetime = new DateTimeImmutable('now', $this->getTimezone());
+            if ($value instanceof \DateTimeImmutable || $value instanceof \DateTime) {
+                $value = $value->setTimezone($this->getTimezone());
```

Comment:
In this case you will mutate `DateTime` instance.
Please also add a tests about it.